### PR TITLE
Path table update

### LINKED ISSUE #334: Add path table to 1 hop query
On click of cell, generate plot table for 1 hop queries

Currently the path table is only shown for 2+ hops

### PR DESCRIPTION
**Pull request by gotdairyya**
With this update, you should be able to click the matrix (after a 1-hop query) and it will display a path table with the paths.

Closes #334

### CODE REVIEW

File: src/components/MultiMatrix.vue [hunk 1]
```diff
@@ -781,7 +791,9 @@ export default defineComponent({
           hideToolTip();
           unHoverEdge(matrixElement);
         })
-        .on('click', (event: MouseEvent, matrixElement: Cell) => store.commit.clickCell(matrixElement))
+        .on('click', (event: MouseEvent, matrixElement: Cell) => {
+          store.commit.clickCell(matrixElement);
+        })
```
Comment (JackWilb) [inline src/components/MultiMatrix.vue:796]: This removes your on click logic you added below from the cells when they update again.

Comment (gotdairyya) [reply src/components/MultiMatrix.vue:796]: Oh so I should add the logic in both parts right?

Comment (JackWilb) [reply src/components/MultiMatrix.vue:796]: I believe so. I think not having it both places was causing it to be removed when the network updated for me.

File: src/components/PathTable.vue [hunk 2]
```diff
@@ -131,6 +137,10 @@ export default defineComponent({
           });
         }
       });
+      // // Removes extra edge in even paths
+      // if (pathLength.value % 2 === 0) {
+      //   toReturn.pop();
+      // }
```
Comment (JackWilb) [inline src/components/PathTable.vue:143]: Looks like a remnant we can remove
```suggestion
```

**Review by JackWilb — CHANGES_REQUESTED** [review]
Just one thing I saw. Everything else looks ready to go!

**Review by JackWilb — APPROVED** [review]

**Pull request merged**